### ISSUE TITLE
Question: LoRA weight specifications

### ISSUE THREAD

**Issue by gryphus-crux** [hako-mikan/sd-webui-supermerger#57]
Hello hako-mikan,
Thank you always for your work, it's one of the only tools that let people use LoRA with specific blocks.
I wanted to ask clarification on what you wrote here:

"Enter LoRA name1:ratio1:block1,LoRA name2:ratio2:block2,..."

Say that on my txt2img tab with your LoRA Block Weight extension I have:
X: values, 0.15
Y: Block ID, IN05-OUT05
Z: Original Weights, ALL0.7 

And ALL0.7 is defined below as 0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7,0.7 on the Weights Setting window.

To combine my LoRA with this specific setting using Supermerger, what is the correct syntax?
"LoRA name1:ratio1:block1" doesn't specify what is the ratio1 referring to. Does this refer to 0.7 weighting of the LoRA? Or the 0.15 weighting of the block? 

Thank you for your help!

**Comment by hako-mikan** [comment]
Its same to LoRA syntax in txt2img.
If you use LoRA in txt2img tabs `<lora:yourlora:1:ALL0.7>`
```
LoRA name1 = yourlora
ratio1 = 1
block1 = ALL0.7
```
If you want to get same result to txt2img tab, check same to strength option.

**Comment by gryphus-crux** [comment]
Thank you for the reply!
I understand that part - if I did <lora:yourlora:1:ALL0.7>, I would get 
[0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7] applied to all the blocks (from BASE-OUT11, i.e. BASE,IN01,IN02,IN04,IN05,IN07,IN08,M00,OUT03,OUT04,OUT05,OUT06,OUT07,OUT08,OUT09,OUT10,OUT11)
That is what's showing on the cmd window too.

But what I'm trying to achieve is this:
[0.7, 0.7, 0.7, 0.7, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.15, 0.7, 0.7, 0.7, 0.7, 0.7, 0.7]

And I've tried this:

loraname:0.7,0.7,0.7,0.7,0.15,0.15,0.15,0.15,0.15,0.15,0.15,0.7,0.7,0.7,0.7,0.7,0.7:BASE,IN01,IN02,IN04,IN05,IN07,IN08,M00,OUT03,OUT04,OUT05,OUT06,OUT07,OUT08,OUT09,OUT10,OUT11

loraname:0.7:BASE,loraname:0.7:IN01,loraname:0.7:IN02,loraname:0.7:IN04,loraname:0.15:IN05,loraname:0.15:IN07,loraname:0.15:IN08,loraname:0.15:M00,loraname:0.15:OUT03,loraname:0.15:OUT04,loraname:0.15:OUT05,loraname:0.7:OUT06,loraname:0.7:OUT07,loraname:0.7:OUT08,loraname:0.7:OUT09,loraname:0.7:OUT10,loraname:0.7:OUT11

loraname:0.7:BASE-IN04,loraname:0.15:IN05-OUT05,loraname:0.7:OUT06-OUT11

None of them seem to work. Is this a feature unimplemented, or is there a different way to go about it?
